### PR TITLE
Milestone3

### PR DESCRIPTION
**Pull request by shloch**
for code review
- Designed the UI of the calculator app ( ``` src/index.css ``` )
Enabling FLEX displays, some default prop values necessary too alter designs

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
+
 class Button extends Component {
   render() {
+    const wideStyle = (this.props.wide) ? 'wideStyle' : '';
+    let bgColor;
+    if (!this.props.color) {
+      bgColor = defaultBgColor;
+    } else {
+      bgColor = { backgroundColor: this.props.color };
+    }
```
Comment (BalintMendli) [inline src/components/Button.js]: Let's set defaultProps for the color prop and then we don't need this logic.
Also set color's propType.

File: src/components/Button.js [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
     return (
-      <div>
-        <button>{this.props.name}</button>
-      </div>
+      <button className={`${wideStyle}`} id='button-component' style={bgColor}>
```
Comment (BalintMendli) [inline src/components/Button.js]: The template literal is unnecessary here. Could be just `className={wideStyle}`.

File: src/components/ButtonPanel.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ class ButtonPanel extends Component {
     return (
       <div id="buttonPanel-component">
         <div id='group-1'>
-          <Button name='AC' />
+          <Button name='AC' color='' wide='' />
```
Comment (BalintMendli) [inline src/components/ButtonPanel.js]: Let's set the colors for buttons that does not have the default orange color.

**Review by BalintMendli — CHANGES_REQUESTED** [review]
Hi @shloch ,

The calculator looks great. 👏 

We just need to make some changes regarding Button's props.

Please check my comments and update the app.

**When done please submit another review request.**

_As always if you have any questions let me know on Slack._

**Comment by shloch** [comment]
@zenott thanks for the reviews, learning a lot from them :)

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hello @shloch 👋 
This is looking Good now, please just fix the linting error, lets just have stickler green.

**request another review when it's done**

**Review by SunnyAsar — APPROVED** [review]
# Hi @shloch 👍 
You had it fixed before i finished my review.
Good Work.

# You May pass 👍

**Pull request merged**